src/twisted/test/test_twistd.py

```diff
@@ -865,6 +901,91 @@ def fakeSetupEnvironment(self, chroot, rundir, nodaemon, umask,
             ['/foo/chroot', '/foo/rundir', True, 56, '/foo/pidfile'])
 
 
+    def test_shedPrivileges(self):
+        """
+        """
+        def switchUIDPass(euid, uid, gid):
```

Comment:
This should contain code that allows you to assert it's been called.

Reply:
Fixed